### ISSUE TITLE
Axe Culling Blade no longer has any mention of piercing Shallow Grave/False Promise/Borrowed Time/etc

### ISSUE THREAD

**Issue by natesampo** [jeffhill/Dota2#14]
Reading the ability text on Axe's Culling Blade implies that it is a pure damage nuke with additional effects if you kill an enemy with it. It does not mention that any enemies below the HP threshold will die as a result, regardless of being affected by a death-preventing ability.

The current text of Culling Blade makes it seem like it works like any other pure damage nuke (QoP Sonic Wave for example), when in reality it has an additional unspoken effect.

**Comment by ScoottMcDoot** [comment]
Alt-text says 'Culling Blade death cannot be prevented by other abilities such as Shallow Grave'. Is this true in your client? What would you like it to say?

**Comment by natesampo** [comment]
I didn't know about the alt-text! That being said, Abaddon's Borrowed Time reads "All damage dealt to you will heal instead of harm", so with the current alt-text on Culling Blade its unclear why Abaddon would heal if his HP were over the damage threshold but die while under it.

**Comment by JacobJustice** [comment]
Culling Blade is HP removal while they are underneath the damage threshold, which is seperate from damage, hence Abaddon dies and doesn't get healed. If they are above the threshold it simply does the damage threshold value as pure damage, which is why it heals Abaddon.

**Comment by natesampo** [comment]
@JacobJustice Yes I understand why it works that way, but the issue is the ability doesn't say that.

**Comment by JacobJustice** [comment]
Gotcha, I think a better description wouldn't say "Axe spots a weakness and strikes, dealing pure damage." It would say "Axe spots a weakness and strikes, instantly killing enemies below a/the threshold. If they are above the threshold, pure damage is dealt instead." and then the rest of the description as is.

**Comment by shaileshgarg** [comment]
Not an issue, this can be closed.
